### PR TITLE
Documentation Changes

### PR DESCRIPTION
**Pull request by diegolovison**

### CODE REVIEW

**Comment by galderz** [comment]
@tristantarrant Do you want to improve on the second change? Or shall we just fix it for now? We now want to promote the use of named caches.

**Comment by galderz** [comment]
@diegolovison Have you verified these changes work fine with latest Infinispan 9.2.0.CR1?

File: tutorials/embedded/step-1.html.haml [hunk 1]
```diff
@@ -42,7 +42,7 @@ title: Embedded Step-by-Step Tutorial
         %code.java
           = preserve do
             :escaped
-              CacheManager cacheManager = new DefaultCacheManager();
```
Comment (diegolovison) [inline tutorials/embedded/step-1.html.haml]: [Here](https://github.com/diegolovison/example-infinispan/blob/b7bf0faca266249761d9f381e1ee42581a40d986/src/main/java/com/github/diegolovison/example/infinispan/EmbeddedCacheManagerExample.java#L24) is the example

File: tutorials/embedded/step-5.html.haml [hunk 2]
```diff
@@ -28,7 +28,7 @@ title: Embedded Step-by-Step Tutorial
             :escaped
               GlobalConfigurationBuilder global = GlobalConfigurationBuilder.defaultClusteredBuilder();
               global.transport().clusterName("WeatherApp");
-              cacheManager = new DefaultCacheManager(global.build(), config.build());
```
Comment (diegolovison) [inline tutorials/embedded/step-5.html.haml]: [Here](https://github.com/diegolovison/example-infinispan/blob/b7bf0faca266249761d9f381e1ee42581a40d986/src/main/java/com/github/diegolovison/example/infinispan/ClusteredCacheManagerExample.java#L27) is the example

**Comment by galderz** [comment]
Thanks @diegolovison

**Pull request merged**